rats-corim/build.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright 2024 Oxide Computer Company
+#
+
+set -o errexit
+set -o pipefail
+
+ROOT=$(cd "$(dirname "$0")" && pwd)
+. "$ROOT/../lib/common.sh"
+
+ARTEFACT="$ROOT/artefact"
+mkdir -p "$ARTEFACT"
+WORK="$ROOT/work"
+mkdir -p "$WORK"
+SRC64="$WORK/src64"
+mkdir -p "$SRC64"
+PROTO="$WORK/proto"
+mkdir -p "$PROTO"
+TOOLS="$WORK/tools"
+mkdir -p "$TOOLS"
+
+NAM='rats-corim'
+REPO="https://github.com/oxidecomputer/$NAM.git"
+
+if [[ -x /usr/gcc/10/bin/gcc ]]; then
+	GCC_DIR=/usr/gcc/10/bin
+elif [[ -x /opt/gcc-10/bin/gcc ]]; then
+	GCC_DIR=/opt/gcc-10/bin
+else
+	fatal "Could not find GCC in any expected location"
+fi
+info "using $GCC_DIR/gcc: $($GCC_DIR/gcc --version | head -1)"
+info "using $GCC_DIR/g++: $($GCC_DIR/g++ --version | head -1)"
```

Comment:
gcc / g++ isn't used to compile `rats-corim` so these checks aren't necessary (it builds fine w/o them for me)